admin_guide/high_availability.adoc

```diff
@@ -11,37 +11,53 @@
 toc::[]
 
 == Overview
-This topic describes how to set up highly-available services on your
-{product-title} cluster.
+This topic describes setting up high-availability for pods and services on your {product-title} cluster.
```

Comment:
I think we can drop the hyphenation here. We don't use it in the topic title, for instance.